docs/samples/contact-center/app.js

```diff
@@ -207,6 +208,7 @@ function logoutAgent() {
 
     setTimeout(() => {
       logoutAgentElm.classList.add('hidden');
+      agentLogin.selectedIndex = 0;
```

Comment:
Needed in order tor reset the login dropdown status as in the register we set one of the options, if the agent is already logged.